### PR TITLE
Update Rank Systems to Version 0.50.2-SNAPSHOT

### PR DESCRIPTION
**Pull request by IllianiCBT**
Incremented the rank systems version in ranks.xml from 0.50.1-SNAPSHOT to 0.50.2-SNAPSHOT. This stops IDEA from pestering us to update it any time we make a change in mhq.

### CODE REVIEW

File: MekHQ/userdata/data/universe/ranks.xml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<rankSystems version="0.50.1-SNAPSHOT">
+<rankSystems version="0.50.2-SNAPSHOT">
```
Comment (HammerGS) [inline MekHQ/userdata/data/universe/ranks.xml:2]: Shouldn't this be 50.02

Comment (IllianiCBT) [reply MekHQ/userdata/data/universe/ranks.xml:2]: So, yes, but not according to our versioning. I think Tex might need to take a look at that.

Comment (HammerGS) [reply MekHQ/userdata/data/universe/ranks.xml:2]: I just fixed it :)
image

Comment (IllianiCBT) [reply MekHQ/userdata/data/universe/ranks.xml:2]: Perfect! I'll get this PR updated when I'm home :)

**Pull request merged**